providers/implementations/keymgmt/ec_kmgmt.c

```diff
@@ -358,8 +358,9 @@ int ec_export(void *keydata, int selection, OSSL_CALLBACK *param_cb,
 
     if ((selection & OSSL_KEYMGMT_SELECT_DOMAIN_PARAMETERS) != 0) {
         bnctx = BN_CTX_new_ex(ec_key_get_libctx(ec));
+        if (bnctx == NULL)
+            goto err;
```

Comment:
I think you need to set `ok = 0` before `goto err`? (see line 362 on LHS).

Reply:
Ah, ok.  I reviewed the changes commit by commit and somehow I overlooked the last one.